databay/planners/aps_planner.py

```diff
@@ -194,4 +198,10 @@ def running(self):
         return self._scheduler.state == STATE_RUNNING
 
     def __repr__(self):
-        return 'APSPlanner(threads:%s)' % (self._threads)
+        return 'ApsPlanner(threads:%s)' % (self._threads)
+
+
+class APSPlanner(ApsPlanner): # pragma: no cover
+    def __init__(self, *args, **kwargs):
+        warnings.warn('APSPlanner was renamed to ApsPlanner and will be permanently changed in version 1.0', DeprecationWarning)
```

Comment:
Might be worth also mentioning what version this change was brought in on in the warning. 
i.e.
`warnings.warn('APSPlanner was renamed to ApsPlanner in version 0.1.x and will be permanently changed in version 1.0', DeprecationWarning)`